CHANGELOG.md

```diff
@@ -6,9 +6,14 @@ librdkafka v1.8.2 is a maintenance release.
 
  * The `librdkafka.redist` 1.8.0 package had two flaws:
    - the linux-arm64 .so build was a linux-x64 build.
-   - the included MSVC 140 runtimes for x64 were infact x86.
+   - the included Windows MSVC 140 runtimes for x64 were infact x86.
    The release script has been updated to verify the architectures of
    provided artifacts to avoid this happening in the future.
+ * Prebuilt binaries for Mac OSX Sierra (10.12) and older are no longer provided.
+   This affects [confluent-kafka-go](https://github.com/confluentinc/confluent-kafka-go).
+ * Prebuilt binaries for Mac OSX now contain statically linked OpenSSL v1.1.1l.
+   Previously the OpenSSL version was either v1.1.1? or v1.0.2? depending on
```

Comment:
do you want the `?`'s in there?

Reply:
Yep, wildcards, but I can see how it might look weird if the reader is not the bash shell interpreter.